sources/packages/web/src/store/modules/institution/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTree } from "vuex";
+import { InstitutionService } from "@/services/InstitutionService";
+import {
+  InstitutionLocationState,
+  RootState,
+  InstitutionUserAndAuthDetails,
+  InstitutionLocationsDetails,
+} from "@/types";
+
+export const actions: ActionTree<InstitutionLocationState, RootState> = {
+  initialize(): boolean {
+    console.log("Initializing the institution store...");
+    return true;
+  },
+  getUserInstitutionDetails(context): Promise<InstitutionUserAndAuthDetails> {
+    return new Promise((resolve, reject) => {
+      InstitutionService.shared
+        .getMyInstitutionDetails()
+        .then(resultComment => {
+          context.commit("setmyInstitutionAndUserDetailsState", resultComment);
```

Comment:
I think that "my" should start with upper case like "setMyInstitutionAndUserDetailsState"